tests/testthat/test-eval-time-single-selection.R

```diff
@@ -0,0 +1,242 @@
+# "selecting single eval time" means how functions like `show_best()` will pick
+# an evaluation time for a dynamic metric when none is given. Previously we
+# would find what is in the data and select a time that was close to the median
+# time. This was fine but inconsistent with other parts of tidymodels that do
+# similar operations. For example, tune_bayes has to have a metric to optimize
+# on so it uses the first metric in the metric set and, if needed, the first
+# evaluation time given to the function.
+
+test_that("selecting single eval time - non-survival case", {
+  library(yardstick)
+
+  met_reg <- metric_set(rmse)
+
+  times_1 <- 1 / 3
+  times_2 <- as.numeric(5:4) / 7
+
+  # ----------------------------------------------------------------------------
```

Comment:
```suggestion
 ```

These lines break the file up _more_ than the `testthat()` blocks which should be the higher level of separation.